package.json

```diff
@@ -70,26 +71,29 @@
     "@rollup/plugin-node-resolve": "^11.1.1",
     "@rollup/plugin-typescript": "^8.1.1",
     "@types/chroma-js": "^1.3.5",
-    "@types/jest": "^23.3.14",
+    "@types/jest": "^26.0.24",
     "@types/node": "^10.17.9",
     "@types/promise-polyfill": "^6.0.0",
     "@types/signals": "1.0.1",
     "@types/sprintf-js": "^1.1.2",
     "babel-plugin-array-includes": "^2.0.3",
     "babelrc-rollup": "^3.0.0",
     "cpy-cli": "^3.1.0",
-    "jest": "^25.5.1",
+    "eslint": "^8.5.0",
+    "eslint-plugin-import": "^2.25.3",
```

Comment:
This brings in eslint (and some plugins) but I'm not sure we have any eslint config or reference to them anywhere else?

Reply:
TBH I'm not sure why I needed that - might have just been required by my own use of it. Fine to omit if things build.

Reply:
It did used to be a dependency, but I think the config was all for the pre-TS code (My PR 882 removed it and I did make a start at sorting out the linting/formatting workflow but it had turned into a massive time-sink and wasn't very high priority - I might have the appetite to have another go at some point :) ).